ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -353,27 +355,56 @@ def prepare_lane_for_export(self, lane_index):
             withBatchProcessing = True
         )
 
-    def post_process_lane_export(self, lane_index):
+    def post_process_lane_export(self, lane_index, time=1):
```

Comment:
Could you add a comment about why you added the `time` parameter and why does it starts at 1? It seems a bit counter-intuitive to me. You're also changing the name of a function that is called from the `dataExportApple` but you probably already thought of this.